docs/interfaces-and-inheritance.md

```diff
@@ -70,14 +70,21 @@ class GetTodosArgs extends PaginationArgs {
 
 This technique also works with input type classes, as well as with object type classes:
 ```typescript
-// `Person` is the object type class we've created earlier in this docs
+@ObjectType()
+class Person {
+  @Field()
+  age: number;
+}
+
 @ObjectType()
 class Student extends Person {
   @Field()
   universityName: string;
 }
 ```
 
+Note that both the subclass and the parent class must be decorated with the `@ObjectType()` decorator for the parent class fields to be picked up on the subclass.
```

Comment:
Note that both the subclass and the parent class must be decorated with the the same class decorator (e.g. `@InputType()` -> `@InputType()`) for the parent class fields to be picked up on the subclass. This means that you can't create e.g. input type class that inherits from object type class.

Reply:
Updated to reflect this as well